core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -383,7 +383,14 @@ public MasterClientContext getMasterClientContext() {
   }
 
   /**
-   * @return the {@link ClientContext} backing this {@link FileSystemContext}
+   * Returns the client context for this file system context.
+   * <p>
+   * Returns an object of type {@link alluxio.ClientContext}
+   * containing only the information required and pertaining to
+   * making network connections and performing operations with remote
```

Comment:
I would fix just this part to: "...containing only the information required to create network connections and perform remote operations on Alluxio processes."

Reply:
Fixed.